devops/cloud-functions/gcf-pin-ipfs-hash/index.js

```diff
@@ -94,8 +117,11 @@ const parseIncomingData = data => {
     } else {
       console.log('No IPFS media hashes found in listing data')
     }
-  } else if (eventName === 'IdentityUpdated') {
-    console.log(`Processing event ${eventName}`)
+  } else if (offerEvents.includes(eventName)) {
+    console.log(`Processing offer event ${eventName}`)
+    hashesToPin.push(getIpfsHashFromBytes32(data.event.returnValues.ipfsHash))
+  } else if (identityEvents.includes(eventName)) {
+    console.log(`Processing identity event ${eventName}`)
     hashesToPin.push(getIpfsHashFromBytes32(data.event.returnValues.ipfsHash))
```

Comment:
Nit: Perhaps doesn't need to be in the if/else blocks.

Reply:
done